src/shared/utils/daoUtils/daoUtils.ts

```diff
@@ -68,7 +68,9 @@ class DaoUtils {
             return plugin.name;
         }
 
-        return this.parsePluginSubdomain(plugin.subdomain);
+        // Fallback chain: subdomain → slug → interfaceType
+        const pluginIdentifier = (plugin.subdomain ?? plugin.slug) || plugin.interfaceType;
+        return this.parsePluginSubdomain(pluginIdentifier);
```

Comment:
`parsePluginSubdomain` expects name in snake-case format, and as I see, only subdomain is in this format. Slugs are just flat and interfaceType is in camel case format.

Why using slug at all? ...interface type is always present, right?

Can we just do something like this:

```suggestion
if (plugin.subdomain) {
        return this.parsePluginSubdomain(plugin.subdomain);
}        

return plugin.interfaceType;
```

Reply:
I added proper formatting for interfaceType to format to the same result as it does for subdomain so

```
subdomain: 'lock-to-vote' -> Lock To Vote
interfaceType: 'lockToVote' -> Lock To Vote
```